static/js/narrow.js

```diff
@@ -91,6 +91,10 @@ exports.set_compose_defaults = function (opts) {
     if (single.has('pm-with')) {
         opts.private_message_recipient = single.get('pm-with');
     }
+
+    if (single.has('huddle-with')) {
+        opts.private_message_recipient = single.get('huddle-with');
+    }
```

Comment:
I think this change shouldn't be here -- because `huddle-with` is not a fully specified narrow (a user could be in multiple huddles along with a single other user).